### ISSUE TITLE
Normalize into square

### ISSUE THREAD

**Issue by hrst** [tancredi/fantasticon#588]
How can I normalize my icon font to a square? Some of my icons have a higher width than height, some have a higher height than width. I'm trying to generate the icon font in such a way that all icons can be used in a square fashion. Currently this does not work, as the normalization seems to bring them all to the same height while ignoring the width. This leads to some icons being "too big". Is there any setting to normalize to a square?

I only found https://github.com/tancredi/fantasticon/issues/328 which seems to be exactly the opposite problem.

In other words, the current behavior for normalize is that the icon "covers" the square, always going the maximum height. I would like the square to "contain" the icon, meaning it should not have a higher width than the maximum height.

Version: 3.0.0
